src/main/java/net/cosmocat/marketplace/database/dto/response/ErrorResponse.java

```diff
@@ -0,0 +1,56 @@
+package net.cosmocat.marketplace.database.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "RFC 9457 compliant error response")
+public class ErrorResponse {
```

Comment:
Aha, catched you - we have ProblemDetail class - the class that I mentioned in my lecture 
<img width="500" height="500" alt="image" src="https://github.com/user-attachments/assets/d79fe3ac-6c7f-48ef-900a-1778a7aa940e" />